### PR TITLE
main/perl-io-compress: upgrade to 2.086

### PR DESCRIPTION
**Pull request by timlegge**

### CODE REVIEW

File: main/perl-io-compress/APKBUILD [hunk 1]
```diff
@@ -1,38 +1,31 @@
...
 makedepends="perl-dev"
-source="http://search.cpan.org/CPAN/authors/id/P/PM/PMQS/${_pkgname}-${pkgver}.tar.gz"
-builddir="$srcdir/$_pkgname-$pkgver"
-
-prepare() {
-	default_prepare
-	export CFLAGS=$(perl -MConfig -E 'say $Config{ccflags}')
-	PERL_MM_USE_DEFAULT=1 perl Makefile.PL INSTALLDIRS=vendor
-}
+subpackages="$pkgname-doc"
```
Comment (Ikke) [inline main/perl-io-compress/APKBUILD]: Are there makedepends missing? In the CI build, no docs seem to be generated.

**Comment by timlegge** [comment]
Strange, not really sure why I submitted this one now.  Other than the changes to the format of the APKBUILD it doesn't upgrade the version.  I am fine with closing...

**Comment by Ikke** [comment]
Lets wait to see what rnalrd says.

**Comment by rnalrd** [comment]
Other than the license update which is okay, all other changes seems to revert to the old-style perl APKBUILDs.

Not merging.

**Comment by timlegge** [comment]
Is there a new format that should be added to apkbuild-cpan so new packages
get the right format?

**Comment by rnalrd** [comment]
apkbuild-cpan already produce the latest version afaik

Il giorno 29 apr 2019, 12:55, alle ore 12:55, Timothy Legge <notifications@github.com> ha scritto:

**Pull request closed without merging**